GWPEstimator.ViewModel/ViewModel/MainWindowViewModel.cs

```diff
@@ -388,13 +388,20 @@ private void Calculate()
                                     IsSteConHyb = true;
                                     IsRc = false;
                                 }
-                                else
+                                else if(string.Equals(Output, "Concrete"))
                                 {
                                     IsConWoodHyb = false;
                                     IsWood = false;
                                     IsSteConHyb = false;
                                     IsRc = true;
                                 }
+                                else
+                                {
+                                    IsConWoodHyb = false;
+                                    IsWood = false;
+                                    IsSteConHyb = false;
+                                    IsRc = false;
+                                }
```

Comment:
I suppose this will not display any material types now when if(optionA) is running correct?